### PR TITLE
Sebin/task/#2284 upgrade cypress to the latest ricardo

### PR DESCRIPTION
**Pull request by corrideat**

### CODE REVIEW

**Comment by taoeffect** [comment]
Interesting, it seems this had no effect on the issue @SebinSong's running into.

File: test/cypress/integration/group-paying.spec.js [hunk 1]
```diff
@@ -346,8 +346,14 @@ describe('Group Payments', () => {
...
       cy.getByDT('payRecord').find('tbody').children().should('have.length', 1)
-      cy.getByDT('payRow').eq(0).find('input[data-test="amount"]').should('have.value', '100')
-      cy.getByDT('payRow').eq(0).find('input[data-test="amount"]').clear({ force: true }).type('50')
+      cy.getByDT('payRow').eq(0).find('input[data-test="amount"]').as('firstRowAmountInput')
+
+      cy.get('@firstRowAmountInput').should('have.value', '100')
+      cy.get('@firstRowAmountInput').clear({ force: true })
+      // NOTE: Since this amount input element has a floating element ('$' sign) above it, cypress thinks the <input /> is not visible, which is not true.
+      //       So we need to add { force: true } here.
+      cy.get('@firstRowAmountInput').type('50', { force: true })
```
Comment (SebinSong) [inline test/cypress/integration/group-paying.spec.js:355]: I can check if this block can also be replaced with `type('{selectall}{del}50')` once this PR is merged.

**Review by SebinSong — APPROVED** [review]
Looks good to me. Amazing work @corrideat !

**Review by taoeffect — APPROVED** [review]
Amazing work @SebinSong & @corrideat !!

**Pull request merged**